templates/detailItems/adr.html

```diff
@@ -1,17 +1,27 @@
 <select ng-model="ctrl.data.meta.type[0]">
 	<option ng-repeat="option in ctrl.availableOptions" value="{{option.id}}">{{option.name}}</option>
 </select>
-<label for="details-pobox">Post office box:</label>
-<input type="text" id="details-pobox" name="address" ng-model="ctrl.data.value[0]" value="" />
-<label for="details-address">Street address:</label>
-<input type="text" id="details-address" name="address" ng-model="ctrl.data.value[2]" value=""
+<!--<label for="details-pobox">Post office box:</label>
+<input type="text" id="details-pobox" name="address" ng-model="ctrl.data.value[0]" value="" />-->
```

Comment:
@jancborchardt we shall not remove the post office box - it's an valid part of the address